### PR TITLE
Add netinfo API (harmful)

### LINKED ISSUE #117: Network Information API
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Network Information API
* Specification or proposal URL: https://wicg.github.io/netinfo/
* Caniuse.com URL (optional): https://caniuse.com/#feat=netinfo
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): @martinthomson, @marcoscaceres, @tomrittervg

### Other information

This API ships in Chrome.  Firefox had an implementation of the API, which has subsequently been removed.  [This dev.platform thread from 2016](https://groups.google.com/d/msg/mozilla.dev.platform/lCZmhCDGHPY/hq3CgPkFDwAJ) covers this.  Some of that discussion is might be stale as the spec has changed a little in the meantime.

Of note, since those discussions, the `Save-Data` feature was moved to this specification.  This is a signal to a site that the client prefers a trade-off in favour of network efficiency when there are choices to be made between the number of bytes transferred and other things (like CPU overheads, quality, etc...)

### PR DESCRIPTION
**Pull request by martinthomson**
Closes #117.

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -419,6 +419,18 @@
     "title": "Native File System",
     "url": "https://wicg.github.io/native-file-system"
   },
+  {
+    "ciuName": "netinfo",
+    "description": "The Network Information API enables web applications to access information about the network connection in use by the device.",
+    "id": "netinfo",
+    "mozBugUrl": null,
```
Comment (marcoscaceres) [inline activities.json:426]: Maybe https://bugzilla.mozilla.org/show_bug.cgi?id=1057169 ?

Comment (marcoscaceres) [reply activities.json:426]: we could "won't fix" that also.

Comment (martinthomson) [reply activities.json]: Thanks for the link.  We probably should, but that's a separate decision.

Comment (annevk) [reply activities.json]: I couldn't resist.

**Pull request merged**